proto/v1/vulnerability.proto

```diff
@@ -171,25 +171,33 @@ message Details {
   bool fix_available = 9;
 }
 
-// This message wraps a location affected by a vulnerability and its
-// associated fix (if one is available).
+// This message wraps a package, version and cpe_uri affected by a vulnerability
+// and its associated fix (if one is available).
 message PackageIssue {
-  // Required. The location of the vulnerability.
-  VulnerabilityLocation affected_location = 1;
-
-  // The location of the available fix for vulnerability.
-  VulnerabilityLocation fixed_location = 2;
-}
-
-// The location of the vulnerability.
-message VulnerabilityLocation {
   // Required. The CPE URI in [cpe format](https://cpe.mitre.org/specification/)
   // format. Examples include distro or storage location for vulnerable jar.
-  string cpe_uri = 1;
+  string affected_cpe_uri = 1;
 
   // Required. The package being described.
-  string package = 2;
+  string affected_package = 2;
 
-  // Required. The version of the package being described.
-  grafeas.v1.package.Version version = 3;
+  // Required. The affected version of the vulnerability.
+  grafeas.v1.package.Version affected_version = 3;
+
+  // The CPE URI in [cpe format](https://cpe.mitre.org/specification/)
+  // format. Examples include distro or storage location for vulnerable jar.
+  // This field will only be populated if different from affected_cpe_uri.
```

Comment:
I think we should still populate this field even if it's same as the affected CPE.